doc/060_forget.rst

```diff
@@ -282,3 +317,49 @@ last-day-of-the-months (11 or 12 depends if the 5 weeklies cross a month).
 And finally 75 last-day-of-the-year snapshots. All other snapshots are
 removed.
 
+Customize pruning
+*****************
+
+To understand the custom options, we first explain, how the pruning process is
+working:
```

Comment:
"how the pruning process works"

Reply:
I'd remove the comma after "explain" too.

Reply:
changed.